src/fairseq2/generation/text.py

```diff
@@ -153,13 +153,15 @@ class TextTranslator:
     _converter: SequenceToTextConverter
     _pad_idx: int
     _source_text_encoder: TextTokenEncoder
+    max_src_len: Optional[int]
```

Comment:
Nit: Could you please name this `_max_source_len` to be consistent with the rest of the naming convention?